mockgen/reflect.go

```diff
@@ -72,7 +73,7 @@ func Reflect(importPath string, symbols []string) (*model.Package, error) {
 		}
 
 		// Build the program.
-		cmd := exec.Command("go", "build", "-o", progBinary, progSource)
+		cmd := exec.Command("go", "build", *buildFlags, "-o", progBinary, progSource)
```

Comment:
I believe that this line is not correct. If buildFlags is empty, you will get an error like:

```
can't load package: package -o: cannot find package "-o" in any of:
/usr/local/go/src/-o (from $GOROOT)
/go/src/-o (from $GOPATH)
```

I think you're missing some quotes here or something.